packages/vscode-dependencies-validation/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+//@ts-check
```

Comment:
dont use a new generated webpack config.
build upon the existing one in app-studio-toolkit for consistency.

Reply:
Done